hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -535,12 +563,11 @@ spec = parallel $ do
               withHydraNode aliceSk [] chain $ \n1 -> do
                 send n1 Init
                 waitUntil [n1] $ HeadIsInitializing testHeadId (fromList [alice])
-                simulateCommit chain (alice, utxoRef 1)
```

Comment:
why?

Reply:
it is not needed since we only look at the `HeadIsInitializing` logs